app/src/main/res/values-it/strings.xml

```diff
@@ -1,23 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="launch_dialog_content">Benvenuto su Extras for Android Auto, un tema per cambiare l\'aspetto di Android Auto
-        \n\n Se vuoi ringraziarmi per il lavoro, puoi farlo con una donazione, dal bottone sotto.
+    <string name="previews_text">🖼 Anteprime! 🖼</string>
+    <string name="launch_dialog_title" translatable="false">Ciao!</string>
+    <string name="launch_dialog_content">Benvenuto in Extras for Android Auto, un tema per cambiare l\'aspetto di Android Auto
+        \n\nSe vuoi ringraziarmi per il lavoro, puoi farlo con una donazione, con il pulsante qui sotto.
         \n Anche una piccola somma sarebbe molto apprezzata 😊</string>
-    <string name="do_not_show_again">Non mostrare di nuovo</string>
+    <string name="do_not_show_again">Non visualizzare di nuovo</string>
```

Comment:
This takes unnecessary extra space to say the same thing.

Reply:
"mostrare" should be not corerct and old style.
Most of major sofwtare house use "visualizzare"
But you could leave it untocuched.